classes/services/PKPNavigationMenuService.inc.php

```diff
@@ -278,7 +278,7 @@ function getDisplayStatus(&$navigationMenuItem, &$navigationMenu) {
 					$navigationMenuItem->setUrl($dispatcher->url(
 						$request,
 						ROUTE_PAGE,
-						$contextPath,
+						'index',
```

Comment:
This change renders all [those](https://github.com/pkp/pkp-lib/blob/main/classes/services/PKPNavigationMenuService.inc.php#L270-L277) unnecessary.